src/devices/quhwa.c

```diff
@@ -0,0 +1,73 @@
+/* Quhwa
+ *
+ * Tested devices:
+ * QH-C-CE-3V (which should be compatible with QH-832AC),
+ * also sold as "1 by One" wireless doorbell
+ *
+ * Copyright (C) 2016 Ask Jakobsen
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ */
+#include "rtl_433.h"
+#include "pulse_demod.h"
+#include "data.h"
+#include "util.h"
+
+
+static int quhwa_callback(bitbuffer_t *bitbuffer) {
+	bitrow_t *bb = bitbuffer->bb;
+	uint8_t *b = bb[0];
+	    
+	b[0] = ~b[0];
+	b[1] = ~b[1];
+	b[2] = ~b[2];
+	
+	unsigned bits = bitbuffer->bits_per_row[0];
+
+	if ((bits == 18) &&
+	    ((b[1] & 1) && (b[1] & 2))  && // Last two bits of second byte are one
+	    (b[2]==0xFF)) // Last two bits are one
```

Comment:
I see, since you inverted the last byte the remaining unused 0 bits are set to 1 too.
Maybe this makes the intent more obvious:
((b[1] & 0x03) == 0x03) &&
((b[2] & 0xC0) == 0xC0)